core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/RdsConfigService.java

```diff
@@ -42,99 +42,73 @@
     private ClusterService clusterService;
 
     @Inject
-    private OrganizationService organizationService;
+    private RdsConnectionValidator rdsConnectionValidator;
 
-    public Set<RDSConfig> retrievePrivateRdsConfigs(IdentityUser user) {
-        return rdsConfigRepository.findForUser(user.getUserId());
+    public Set<RDSConfig> retrieveRdsConfigs() {
+        Organization organization = getOrganizationService().getDefaultOrganizationForCurrentUser();
+        return rdsConfigRepository.findAll(organization.getId());
     }
 
-    public RDSConfig getPrivateRdsConfig(String name, IdentityUser user) {
-        return Optional.ofNullable(rdsConfigRepository.findByNameInUser(name, user.getUserId()))
-                .orElseThrow(notFound("RDS configuration", name));
+    public RDSConfig getByName(String name) {
```

Comment:
could be renamed to ```getByNameForDefaultOrg```. also it is used in too many places. should be used only in the v1 controller which falls back to the default org